{"name": "test_login_fail[data1]", "status": "passed", "parameters": [{"name": "data", "value": "{'id': 2, 'title': '7位验证码', 'url': 'http://192.168.3.157:8069/dongwang-app/app/user/login', 'json': {'type': '1', 'flashToken': '', 'deviceId': 'aaa', 'device': '1', 'smsCode': '1231234', 'phone': '113157121093'}, 'expected': {'status_code': 500, 'message': '验证码输入错误,请重新输入'}}"}], "start": 1614052232124, "stop": 1614052232160, "uuid": "46014e72-630b-4ddd-bc19-c80813473377", "historyId": "b6fa7ff97725f1469f0d62024f34a18b", "testCaseId": "62fb4c4c1bc44f1f23ceb53be330b5b0", "fullName": "case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "错误验证码"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "4172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}